### PR TITLE
Precise terminology

### LINKED ISSUE #26539: Incorrect description of |. Bitwise OR not "pipe symbol"
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/GPUCanvasContext/configure

### What specific section or headline is this issue about?

Parameters usage

### What information was incorrect, unhelpful, or incomplete?

Note that multiple possible usages can be specified by separating values with pipe symbols, for example:

### What did you expect to see?

Note that multiple possible usages can be specified using the bitwise OR operator, for example:

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/gpucanvascontext/configure`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/GPUCanvasContext/configure
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/gpucanvascontext/configure/index.md
* Last commit: https://github.com/mdn/content/commit/b6984118ac9482e683a654edfefa4b426ca3c7ca
* Document last modified: 2023-04-07T06:03:34.000Z

</details>

### PR DESCRIPTION
**Pull request by advitiya7**
### Related issues and pull requests

fixes #26539

### CODE REVIEW

**Comment by advitiya7** [comment]
@Josh-Cena i have made the changes

File: files/en-us/web/api/gpucanvascontext/configure/index.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ configure(configuration)
         - `GPUTextureUsage.TEXTURE_BINDING`: The texture can be bound for use as a sampled texture in a shader, for example in a bind group entry in a {{domxref("GPUDevice.createBindGroup()")}} call.
         - `GPUTextureUsage.STORAGE_BINDING`: The texture can be bound for use as a storage texture in a shader, for example in a bind group entry in a {{domxref("GPUDevice.createBindGroup()")}} call.
 
-        Note that multiple possible usages can be specified by separating values with pipe symbols, for example:
+        Note that multiple possible usages can be specified using the bitwise OR operator, for example:
```
Comment (Josh-Cena) [inline files/en-us/web/api/gpucanvascontext/configure/index.md:48]:
```suggestion
        Note that multiple possible usages can be specified using the [bitwise OR operator](/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_OR). For example:
```

**Review by Josh-Cena — APPROVED** [review]

**Comment by advitiya7** [comment]
@Josh-Cena just a general question if you don't mind can you please tell doc reps like mdn where i can contribute?

**Comment by Josh-Cena** [comment]
Sorry, I'm not primarily a technical writer so I don't know what other doc repos there are.

**Pull request merged**